yellowbrick/classifier/rocauc.py

```diff
@@ -85,9 +90,17 @@ class ROCAUC(ClassificationScoreVisualizer):
 
     per_class : bool, default: True
         Plot the ROC curves for each individual class. This should be set
-        to false if only the macro or micro average curves are required. Per-
-        class classification is not defined for binary classification problems
-        with estimators with only a decision_function method.
+        to false if only the macro or micro average curves are required. For true
+        binary classifiers, setting per_class=False will plot the positive class
+        ROC curve, and per_class=True will use ``1-P(1)`` to compute the curve of
+        the negative class if only a decision_function method exists on the estimator.
+
+    binary : bool, default: False
+        This argument quickly resets the visualizer for true binary classification
+        by updating the micro, macro, and per_class arguments to False (do not use
+        in conjunction with those other arguments). Note that this is not a true
+        hyperparameter to the visualizer, it just collects other parameters into
+        a single, simpler argument.
```

Comment:
This metaparameter is just a quick helper for setting micro/macro/per_class to False.